src/js/components/Beacon/VariantsForm.js

```diff
@@ -19,52 +21,57 @@ const CHROMOSOME_REGEX = /^([1-9][0-9]?|X|x|Y|y|M|m|MT|mt)$/;
 const NUCLEOTIDES_REGEX = /^([acgtnACGTN])*$/;
 const DIGITS_REGEX = /^[0-9]+$/;
 
-const FORM_FIELDS = {
-  referenceName: {
-    name: 'Chromosome',
-    rules: [{ pattern: CHROMOSOME_REGEX, message: 'Enter a chromosome name, eg: "17" or "X"' }],
-    placeholder: '1-22, X, Y, M',
-    initialValue: '',
-    type: 'string',
-  },
-  start: {
-    name: 'Variant start',
-    rules: [{ pattern: DIGITS_REGEX, message: 'enter a postion number, eg "100"' }],
-    placeholder: 'eg 100',
-    initialValue: '',
-    type: 'number',
-  },
-  end: {
-    name: 'Variant end',
-    rules: [{ pattern: DIGITS_REGEX, message: 'enter a postion number, eg "200"' }],
-    placeholder: 'eg 200',
-    initialValue: '',
-    type: 'number',
-  },
-  referenceBases: {
-    name: 'Reference base(s)',
-    rules: [{ pattern: NUCLEOTIDES_REGEX, message: 'enter any combination of A, C, G, T, or N' }],
-    placeholder: 'A, C, G, G, T or N',
-    initialValue: '',
-    type: 'string',
-  },
-  alternateBases: {
-    name: 'Alternate base(s)',
-    rules: [{ pattern: NUCLEOTIDES_REGEX, message: 'enter any combination of A, C, G, T, or N' }],
-    placeholder: 'A, C, G, T or N',
-    initialValue: '',
-    type: 'string',
-  },
-  assemblyId: { name: 'Assembly ID', rules: [{}], placeholder: '', initialValue: '' },
-};
-
 const FORM_STYLE = {
   display: 'flex',
   flexDirection: 'column',
 };
 const FORM_ROW_GUTTER = [12, 0];
 
 const VariantsForm = ({ assemblyIdOptions }) => {
+  const { t: td } = useTranslation(DEFAULT_TRANSLATION);
+
+  const FORM_FIELDS = useMemo(
+    () => ({
+      referenceName: {
+        name: td('Chromosome'),
+        rules: [{ pattern: CHROMOSOME_REGEX, message: td('Enter a chromosome name, eg: "17" or "X"') }],
+        placeholder: '1-22, X, Y, M',
+        initialValue: '',
+        type: 'string',
+      },
+      start: {
+        name: td('Variant start'),
+        rules: [{ pattern: DIGITS_REGEX, message: td('enter a postion number, eg "100"') }],
+        placeholder: 'eg 100',
+        initialValue: '',
+        type: 'number',
+      },
+      end: {
+        name: td('Variant end'),
+        rules: [{ pattern: DIGITS_REGEX, message: td('enter a postion number, eg "200"') }],
+        placeholder: 'eg 200',
+        initialValue: '',
+        type: 'number',
+      },
+      referenceBases: {
+        name: td('Reference base(s)'),
+        rules: [{ pattern: NUCLEOTIDES_REGEX, message: td('enter any combination of A, C, G, T, or N') }],
+        placeholder: 'A, C, G, G, T or N',
+        initialValue: '',
+        type: 'string',
+      },
+      alternateBases: {
+        name: td('Alternate base(s)'),
```

Comment:
Translating the `name` property here breaks variant search, since form information is filled in according to the name. Instead of translating `name`, add a new field called "label" and translate that, eg:

```
name: 'Chromosome,
label: td('Chromosome')
```
and then update the label field in `VariantInput` and `AssemblyIdSelect` so that it says `label={field.label}` instead of the current `label={field.name}`. Sorry if I missed this before.

Reply:
done, nw. I translated the label in the final

Reply:
ah, this is a cleaner solution, nice.